interface/modules/zend_modules/public/index.php

```diff
@@ -43,6 +43,9 @@
 if(strtolower($controllerName) == 'soap' && strtolower($actionName) == 'index') {
     $ignoreAuth_offsite_portal = true;
 }
+elseif($_REQUEST['recipient'] === 'patient' && $_REQUEST['site'] && $controllerName){
```

Comment:
Added authentication through a forward of current portal session login authorizations.